frontend/test/app/components/ExperimentDisableDialog-test.js

```diff
@@ -27,8 +28,8 @@ describe('app/components/ExperimentDisableDialog', () => {
 
   it('should render a modal container', () => {
     expect(subject.find('.modal-container')).to.have.property('length', 1);
-    expect(subject.find('.modal-header').props()['data-l10n-args'])
-      .to.equal(JSON.stringify({ title: experiment.title }));
+    expect(findLocalizedById(subject, 'feedbackUninstallTitle').props()['$title'])
```

Comment:
Lots of test updates to account for the change in finding localized nodes... I wonder if in the future (i.e. not retroactively as a nit on this PR) we should just switch to known class names as handles for tests?